phase2/addon/lib/panel/events.js

```diff
@@ -16,11 +16,12 @@ const { emit } = require("sdk/event/core");
 
 let channel = {};
 
-function forward({ subject, type, data })
-  emit(channel, "data", { target: subject, type: type, data: data });
+function forward({ subject, type, data }) {
```

Comment:
in general, eslint hates one line functions without blocks, unless they are arrow functions.  I don't know why